nxdrive/gui/application.py

```diff
@@ -1792,9 +1792,24 @@ def refresh_active_sessions_items(self, dao: EngineDAO, /) -> None:
     def refresh_completed_sessions_items(self, dao: EngineDAO, /) -> None:
         """Refresh the list of completed sessions if a change is detected."""
         sessions = self.api.get_completed_sessions_items(dao)
+        sessions = [self._add_csv_path_to_session(session) for session in sessions]
```

Comment:
Can we just not using that kind of hack? The file name is generated given several knowns parts, it should be generated on the fly and not stored in the database. If we decide to change the file name in a future version, we would have to deal with that. Let's not store the file name at all.